src/StatBlock.cpp

```diff
@@ -374,6 +375,10 @@ bool StatBlock::loadSfxStat(FileParser *infile) {
 		// @ATTR sfx_levelup|filename|Filename of sound effect for leveling up.
 		sfx_levelup = infile->val;
 	}
+	else if (infile->key == "sfx_lowhp") {
```

Comment:
It is in `engine/avatar/"+gfx_base+".txt` to allow consistent sounds for **male** vs **female** characters. It would be no problem with looping beeping, but if you use sounds like loud "aghhh", it would sound silly if you use male voice for female character and vice versa. 

This can be solved by having `sfx_lowhp_male` and `sfx_lowhp_female` + additional logic in code, but I am not sure, that is cleaner.

What do you think, maybe you have better solution?

Reply:
Ah, I wasn't thinking of gender-specific sounds. In that case, your solution is what we want to use.